backend/src/main/scala/bloop/util/Base64Uuid.scala

```diff
@@ -0,0 +1,27 @@
+package bloop.util
+import java.nio.ByteBuffer
+import java.util.{Base64, UUID}
+
+object Base64Uuid {
+
+  // Copied from https://stackoverflow.com/a/29836273
+  implicit class UuidWithBase64(val uuid: UUID) extends AnyVal{
+    def toBase64String: String = {
+      val bb = ByteBuffer.wrap(new Array[Byte](16))
+      bb.putLong(uuid.getMostSignificantBits)
+      bb.putLong(uuid.getLeastSignificantBits)
+      Base64.getUrlEncoder.encodeToString(bb.array)
+    }
+  }
+
+  def apply(base64: String): UUID = {
```

Comment:
Is this being used? If not, let's remove it, we should be never decoding Base 64 to UUID, all we want is a shorter UUID for a random id :smile:

Reply:
Removed